### PR TITLE
Icon: add ri-user-add-fill

### PR DESCRIPTION
**Pull request by ninariccimarie**

### CODE REVIEW

File: src/@next/Icon/icons/icons.ts [hunk 1]
```diff
@@ -164,6 +164,7 @@ export const iconNames = [
   'ri-twitter-line',
   'ri-upload-fill',
   'ri-upload-line',
+  'ri-user-add-fill',
   'ri-user-line',
```
Comment (chiahou) [inline src/@next/Icon/icons/icons.ts:169]: hi @ninariccimarie what about user-add-line? Line style is used in default state, where Fill style appears on hover

Comment (ninariccimarie) [reply src/@next/Icon/icons/icons.ts:169]: @chiahou do we need it? we usually just add what we need

Comment (ninariccimarie) [reply src/@next/Icon/icons/icons.ts:169]: as discussed, we need the user-add-line 👍 I've included it in @chiahou

**Review by aguss787 — APPROVED** [review]
lgmt!

**Review by chiahou — APPROVED** [review]

**Pull request merged**